test_runner/src/main/kotlin/ftl/args/CreateIosArgs.kt

```diff
@@ -33,7 +33,9 @@ private fun createIosArgs(
     testTargets = flank.testTargets?.filterNotNull().orEmpty(),
     obfuscateDumpShards = obfuscate,
     app = gcloud.app?.normalizeFilePath().orEmpty(),
-    testSpecialEntitlements = gcloud.testSpecialEntitlements ?: false
+    testSpecialEntitlements = gcloud.testSpecialEntitlements ?: false,
+    onlyTestConfiguration = flank.onlyTestConfiguration,
```

Comment:
should we not have an .orEmpty here?

Reply:
Agreed, empty and null means the same in this context, so we don't need both, so `onlyTestConfiguration` and `skipTestConfiguration` in IosArgs shouldn't be nullable.